iOS/README.md

```diff
@@ -1,16 +1,72 @@
-### Setup instructions for MMIW AR app.
+# MMIW AR Face Mesh App
```

Comment:
Looks like the wrong README was copied in here 😄 Let's bring back the one for the iOS directory, not the root directory.